sql/catalyst/src/main/scala/org/apache/spark/sql/types/CalendarIntervalType.scala

```diff
@@ -22,11 +22,14 @@ import org.apache.spark.unsafe.types.CalendarInterval
 
 /**
  * The data type representing calendar time intervals. The calendar time interval is stored
```

Comment:
`representing` -> `represents`? @srowen Does `representing` sound good for you?

Reply:
representing sounds right. "[This class is] The data type representing ..."